vendor/github.com/docker/libnetwork/drivers/macvlan/macvlan.go

```diff
@@ -0,0 +1,117 @@
+package macvlan
```

Comment:
instead of making changes under `vendor/` you should make the neccessary changes in the repo and then just bump the commit in `vendor.conf` followed my running the vndr tool (which is installed in the devenv container)

You would need to PR the revert in httsp://github.com/balena-os/balena-libnetwork. And actually make sure to use the `18.09-balena` branch there.